src/bo4e/utils/__init__.py

```diff
@@ -75,5 +75,5 @@ def postprocess_docstring(cls: type[T]) -> type[T]:
     """
     if cls.__doc__ is not None:
         cls.__doc__ = cls.__doc__.format(__gh_version__=__gh_version__)
-    add_comments_to_description(cls)
```

Comment:
do we need this line?

Reply:
Don't be so fast xD https://github.com/bo4e/BO4E-python/pull/949/commits/2a8e914b4e32460fdd0917324b243ab65941a513